lib/index.js

```diff
@@ -593,16 +588,16 @@ api.manageIdentity = brCallbackify(async ({actor, accountId, identityId}) => {
     translate: 'owner'
   });
 
-  const result = await database.collections.identity.update(
-    {id: database.hash(identityId)},
-    {$set: {'meta.bedrock-account.account': accountId}},
-    // FIXME: call update meta on brIdentity or update sequence here
-    // $inc: {'meta.sequence': 1}
-    database.writeOptions);
+  const result = await database.collections.identity
+    .update({id: database.hash(identityId)},
```

Comment:
Previous style was correct.

Reply:
yeah that's what I went with consistent, but paren spacing is really complex.